packages/core/types/test/test.js

```diff
@@ -1,96 +0,0 @@
-"use strict";
```

Comment:
This is an auto-generated (by tsc for testing), `.gitignore`d file so I removed it from source control.